Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocMultilineTextValue.css

```diff
@@ -1,15 +1,27 @@
 /* Style applied to the BocMultilineTextValue. */
 
+:root
+{
+  --min-height: 4.5em;
+  --mozilla-offset: 0.1em;
+}
+
 span.bocMultilineTextValue
 {
   width: 15em;
-  min-height: 4.5em;
+  min-height: var(--min-height);
   position: relative;
   display: inline-block;
   vertical-align: middle;
   overflow: hidden;
```

Comment:
okay, I've just doodled a bit. 

We should remove the overflow (test things still work), make an outline that's 0.2em (perhaps 0.3em, is tbd) and uses the border blue. For readonly, we also use the outline and background color.